modal/secret.py

```diff
@@ -169,7 +169,11 @@ def from_name(
             str
         ] = [],  # Optionally, a list of required environment variables (will be asserted server-side)
     ) -> "_Secret":
-        """Create a reference to a persisted Secret
+        """Reference a Secret by its name.
+
+        In contrast to most other Modal objects, named Secrets must be provisioned
+        from the Dashboard. See other methods for alternate ways of creating a new
```

Comment:
This sentence took me a sec to parse, maybe better: See other methods for alternative ways to create a new Secret from code.